### DISCUSSION TITLE
Orca Slicer g-code preview window

### DISCUSSION THREAD

**Discussion by Merlwynd** [OrcaSlicer/OrcaSlicer#506] [General]
I'm having trouble opening up the g-code window in the preview section of OrcaSlicer 1.5.0. The change notes say to use 'c' shortcut key. But exactly which 'c' shortcut key?...ALT+C, Win+C, Ctrl+ALt+C, etc? None of the combinations I have tried have worked....and g-code window is turned on in Preferences....Has it worked for anyone else?

**Comment by Manfredo1952** [comment]
Was going to comment on that feature as well:    Yes, it does work here (iMac), but in order to see that window in the preview section, you **need to move the lower/ghorizontal  slider to the left a bit**   (might be an idea to amend the description?)  and the shortcut is just c - ![image](https://user-images.githubusercontent.com/56330568/226080664-5108717e-5fda-415a-b4f6-9b8341d24414.png)

**Reply by Merlwynd** [reply]
Thanks...moving the slider slightly left did it..(not sure how you fingered that one out!!). Then Alt+C on Windows toggles it on and off. Thanks so much for the info!!

**Comment by Mario-vminion** [comment]
is it also possible to put the g-code away ... now it stay in the window...

**Reply by SergioSV96** [reply]
![image](https://github.com/SoftFever/OrcaSlicer/assets/20419761/7673bf8b-9ee1-4eae-bd98-8649f1b781ef)

**Reply by Noisyfox** [reply]
Or:
![image](https://github.com/SoftFever/OrcaSlicer/assets/1537155/74d810b2-1fc3-4740-a636-35b443a74ea7)
